addon/templates/components/common/bs-form/element.hbs

```diff
@@ -30,35 +30,12 @@
     (component this.controlComponent
       value=this.value
       id=this.formElementId
-      name=@name
       type=this.controlType
       label=@label
-      placeholder=placeholder
-      autofocus=autofocus
       disabled=disabled
       readonly=readonly
       required=required
-      controlSize=controlSize
-      tabindex=tabindex
-      minlength=minlength
-      maxlength=maxlength
-      min=min
-      max=max
-      pattern=pattern
-      accept=accept
-      autocomplete=autocomplete
-      autocapitalize=autocapitalize
-      autocorrect=autocorrect
-      autosave=autosave
-      inputmode=inputmode
-      multiple=multiple
-      step=step
       form=form
```

Comment:
What about this? Shouldn't this be also removed?

Reply:
You mean the `form`? Is this related to arguments used for HTML attributes only? I thought the instance of `<BsForm>` is passed for some internal communication reasons. But to be honest I haven't had a look at these part of the code for quiet a while.

Reply:
> You mean the form? Is this related to arguments used for HTML attributes only? 

Yeah. I believe so. It is just an ID, and not passed by default AFAICT. And it has been deprecated! :wink:

Reply:
You are right. Removed in 49868eea220dd2ad269fa56b5dc753782c8e2ba9.

What a confusing naming. Now that you pointed it out I remember that it also was totally confused about that one when deprecating it.